core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -445,10 +445,12 @@ class QualificationAppInfo(
           e.children.map(x => x.filterNot(_.isSupported))
         }.flatten
         topLevelExecs ++ childrenExecs
-      }.map(_.exec).toSet.mkString(";").trim
+      }.map(_.exec).toSet.mkString(";").trim.replaceAll("\n", "")
+        .replace(",", ":")
```

Comment:
Do we need to apply that change to other code blocks that do something similar? for example grepping on `replace(",", ":")`.
If we expect we need this processing in different expressions, we can put `.mkString(separator).trim.replaceAll("\n", "").replace(",", ":")` as a method in `ToolUtils.scala`
I see in some other places

Reply:
So I took a quick look and I don't think any of the other ones would just print out things without us already formatting it properly. But yes ideally this gets moved to a more generic place and we stop doing the one of applying things.  And we do something similar in profiling tool. That is a bigger change we can follow up on.

Reply:
I filed an issue to followup